packages/firebase_auth/firebase_auth_platform_interface/lib/src/method_channel/method_channel_firebase_auth.dart

```diff
@@ -23,9 +24,12 @@ class MethodChannelFirebaseAuth extends FirebaseAuthPlatform {
   );
 
   static Map<String, MethodChannelFirebaseAuth>
-      _methodChannelFirebaseAuthInstances =
+      methodChannelFirebaseAuthInstances =
```

Comment:
Is it expected for this to be public? It's public but undocumented

Reply:
I've added documentation, needed to get the Auth instance in the error parsing